requirements.py

```diff
@@ -24,16 +27,15 @@
     "-r",
     "--requirements",
     dest="requirements",
-    default="requirements.txt",
+    default=None,
     help="specify alternative requirements.txt",
 )
 parser.add_argument(
     "-f",
     "--filename",
     dest="requirements_extra",
-    default="requirements_extra.json",
```

Comment:
What is the significance of these default changes?

Reply:
Normally, when you open a file with a relative filename, it opens from the current working directory.
The current working directory is where you run the app from, not where the app is.
I want it so that running this script from any location will make it open the requirements files from the script's location, not the current working directory.
`None` is required so that it can differentiate no args, meaning read from default location, where the script is, from the user manually putting in 'requirements.txt', which will open that file from current working directory.